bitcoinspv/types/supported_backends.go

```diff
@@ -0,0 +1,27 @@
+package types
+
+type (
+	SupportedBtcNetwork string
```

Comment:
Remove the `Btc` from here so we just have `SupportedNetwork` once we move into btc package

Reply:
Renamed